adyenv6core/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -379,9 +386,16 @@ private AddressModel createBillingAddress(PaymentDetailsWsDTO paymentDetails) {
     }
 
     @Override
-    public PaymentsResponse handleRedirectPayload(final String payload) {
+    public PaymentsResponse handleRedirectPayload(HashMap<String, String> details) {
```

Comment:
sometimes details can contain an object is that supported with HashMap<String,String> or should we make it object,object ?

Reply:
details will always contain string, string . It is same as core platform.

Reply:
ok